iOSClient/Data/NCManageDatabase+Metadata.swift

```diff
@@ -232,20 +232,25 @@ extension tableMetadata {
     }
 
     @objc var isDirectoryE2EE: Bool {
-        NCUtility.shared.isDirectoryE2EE(serverUrl: serverUrl, account: account, urlBase: urlBase, userId: userId)
+        NCUtility.shared.isDirectoryE2EE(account: account, urlBase: urlBase, userId: userId, serverUrl: serverUrl)
+    }
+
+    var isDirectoryTopE2EE: Bool {
+        NCUtility.shared.isDirectoryE2EETop(account: account, serverUrl: serverUrl)
```

Comment:
We have `isDirectoryTopE2EE` and `isDirectoryE2EETop` here, let's maybe choose a single way to name this